substrate/frame/staking/stake-tracker/src/mock.rs

```diff
@@ -0,0 +1,537 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![cfg(test)]
+
+use crate::{self as pallet_stake_tracker, *};
+
+use frame_election_provider_support::{ScoreProvider, VoteWeight};
+use frame_support::{derive_impl, parameter_types, traits::ConstU32};
+use sp_runtime::BuildStorage;
+use sp_staking::{Stake, StakingInterface};
+
+pub(crate) type AccountId = u64;
+pub(crate) type Balance = u64;
+
+type Block = frame_system::mocking::MockBlockU32<Test>;
+
+// Configure a mock runtime to test the pallet.
+frame_support::construct_runtime!(
+	pub enum Test
+	{
+		System: frame_system,
+		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
+		VoterBagsList: pallet_bags_list::<Instance1>::{Pallet, Call, Storage, Event<T>},
+		TargetBagsList: pallet_bags_list::<Instance2>::{Pallet, Call, Storage, Event<T>},
+		StakeTracker: crate,
```

Comment:
usually we do `pub use crate as stake_tracker` to make them look a bit more unified.